website/docs/r/webhook_subscription.html.markdown

```diff
@@ -21,6 +21,9 @@ resource "pagerduty_webhook_subscription" "foo" {
   delivery_method {
     type = "http_delivery_method"
     url = "https://example.com/receive_a_pagerduty_webhook"
+    custom_headers = {
+      example_key = "example_value"
+    }
```

Comment:
```suggestion
    custom_headers {
        name = "X-Foo"
        value = "foo"
    }
    custom_headers {
        name = "X-Bar"
        value = "bar"
    }
```

`custom_headers` attribute requires `name` and `value` key/pair for each header, and this aligns with the upstream PD API schema:
https://developer.pagerduty.com/api-reference/b3A6MjkyNDc4NA-create-a-webhook-subscription

Although it feels a bit verbose, users can use the `dynamic` block for more programmable use as usual.